devito/function.py

```diff
@@ -566,6 +566,9 @@ def laplace2(self, weight=1):
         return sum([second_derivative(first * weight, dim=d, order=order)
                     for d in self.space_dimensions])
 
+    # Pickling support
+    _pickle_kwargs = TensorFunction._pickle_kwargs + ['grid']
+
 
 class TimeFunction(Function):
     """
```

Comment:
Will probably need to add `time_order`, `time_dim`, etc here also, I guess?

Reply:
think so. @ggorman ?